### PR TITLE
Add lv_btnm_set_one_toggle/lv_btnm_get_one_toggle functions

### PR DESCRIPTION
**Pull request by embeddedt**
@kisvegabor This implements the behavior we discussed in #954.

### CODE REVIEW

**Comment by kisvegabor** [comment]
Thank you! It looks good in general. I have only a few remarks, see them in the comments.

**Comment by embeddedt** [comment]
@kisvegabor I don't see any comments... did your review get lost somehow?

**Comment by kisvegabor** [comment]
I tagged you.
Here are the comments for me: https://github.com/littlevgl/lvgl/pull/966/files

**Comment by embeddedt** [comment]
@kisvegabor There are no comments there for me. Something really strange is happening. Did you go to the "Review changes" box at the top and click "submit review"? If the comments say "Pending" they won't show up for me till you do that.

**Comment by kisvegabor** [comment]
Oh, I didn't... How have I missed that so far!? It explains a lot of things

File: src/lv_objx/lv_btnm.c [hunk 1]
```diff
@@ -739,6 +765,7 @@ static lv_res_t lv_btnm_signal(lv_obj_t * btnm, lv_signal_t sign, void * param)
                 } else {
                     ext->ctrl_bits[ext->btn_id_pr] |= LV_BTNM_CTRL_TGL_STATE;
                 }
+                make_one_button_toggled(btnm, ext->btn_id_pr);
```
[the remark this answers is not in the document]
Comment (embeddedt) [reply src/lv_objx/lv_btnm.c]: Oops. I didn't catch that, thanks!

[the remark this answers is not in the document]
Comment (embeddedt) [reply src/lv_objx/lv_btnm.c]: Fixed in my latest commit.

File: src/lv_objx/lv_btnm.c [hunk 2]
```diff
@@ -1034,5 +1061,24 @@ static bool maps_are_identical(const char ** map1, const char ** map2)
...
+/**
+ * Enforces a single button being toggled on the button matrix.
+ * It simply clears the toggle flag on other buttons.
+ * @param btnm Button matrix object
+ * @param btn_idx Button that should remain toggled
+ */
+static void make_one_button_toggled(const lv_obj_t *btnm, uint16_t btn_idx)
+{
+	lv_btnm_ext_t * ext = lv_obj_get_ext_attr(btnm);
+
```
[the remark this answers is not in the document]
Comment (embeddedt) [reply src/lv_objx/lv_btnm.c:1075]: I reworked this function to make use of existing `lv_btnm` APIs. Good point.

**Comment by embeddedt** [comment]
@kisvegabor I think everything you mentioned is fixed now.

**Comment by kisvegabor** [comment]
Looks good to me! :+1: 
Thank you!

**Pull request merged**